modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/PlaylistDao.kt

```diff
@@ -29,6 +30,9 @@ abstract class PlaylistDao {
     @Query("SELECT * FROM smart_playlists WHERE manual = 0 AND deleted = 0 AND draft = 0 ORDER BY sortPosition ASC")
     abstract fun observeSmartPlaylists(): Flow<List<SmartPlaylist>>
 
+    @Query("UPDATE smart_playlists SET deleted = 1, syncStatus = ${SYNC_STATUS_NOT_SYNCED} WHERE uuid = :uuid")
+    abstract suspend fun markPlaylistAsDeleted(uuid: String)
```

Comment:
This is just for testing, right? As if the user isn't signed in, we can delete the row.

Reply:
No, this is production code. `PodcastSyncProcess` deletes marked playlists if user isn't signed in.

https://github.com/Automattic/pocket-casts-android/blob/50b2ccbc4870f7f4c800809b71c637ed6b5ce0f1/modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/sync/PodcastSyncProcess.kt#L96-L102

Reply:
Oh I see, thanks for pointing that out.